core/conn/odbc/src/odbc/nsksrvrcore/srvrothers.cpp

```diff
@@ -6378,6 +6378,124 @@ odbc_SQLSrvr_FetchPerf_sme_(
 
 } // odbc_SQLSrvr_FetchPerf_sme_()
 
+extern "C" void
+odbc_SQLSrvr_ExtractLob_sme_(
+    /* In    */ CEE_tag_def objtag_
+  , /* In    */ const CEE_handle_def *call_id_
+  , /* Out   */ odbc_SQLsrvr_ExtractLob_exc_ *exception_
+  , /* In    */ IDL_long extractLobAPI
+  , /* In    */ IDL_string lobHandle
+  , /* Out   */ IDL_long_long &lobDataLen
+  , /* Out   */ IDL_char *& lobDataValue
+  )
+{
+    char LobExtractQuery[1000];
+    char RequestError[200] = {0};
+    SRVR_STMT_HDL  *QryLobExtractSrvrStmt = NULL;
+
+    if ((QryLobExtractSrvrStmt = getSrvrStmt("", TRUE)) == NULL)
```

Comment:
Could the corresponding statements for extract be prepared once with parameters and then executed ?